.github/jobs/ios.yml

```
@@ -1,14 +1,20 @@
+parameters:
+  name: ''
+  vmImage: ''
+  xCodeVersion: ''
+  simulator: ''
+
 jobs:
-- job: iOS
+- job: ${{ parameters.name }}
```

nit: I've been using no spaces in JsRuntimeHost. (A bunch of places)
```suggestion
- job: ${{parameters.name}}
```